### PR TITLE
Split create study modal into separate tabs

### LINKED ISSUE #33: Split study creation in to seperate pages
Currently, all the information for a study is entered on a single page. This means that the input boxes are all smaller than what the user might require. This is especially true for the email input box.

To improve the workflow, the email information input should be moved to another screen following on after you have entered the study details.

### PR DESCRIPTION
**Pull request by jsoulsby**
Resolves #33 

Implemented tabs within the "Create Study" modal that opens on the "+ Create New Study" button in the "Study" page.

I increased the size of the body textbox in the "Invitation Email" tab (as seen below) to an arbitrary 300px to give more space for an email, if this is too small/large please let me know.

image

### CODE REVIEW

**Comment by jsoulsby** [comment]
@Nateeo @Jcen902 Ready to review.

File: src/main/webapp/app/entities/study/study-dialog.component.html [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
+                    </button>
+                    <button type="submit" [disabled]="editForm.form.invalid || isSaving" class="btn btn-primary">
+                        <span class="fa fa-save"></span>&nbsp;
+                        <span>Save</span>
+                    </button>
+                </div>
+            </ng-template>
+        </ngb-tab>
+    </ngb-tabset>
+</form>
```
Comment (Nateeo) [inline src/main/webapp/app/entities/study/study-dialog.component.html]: should have newline at end of file

File: src/main/webapp/content/css/global.css [hunk 2]
```diff
@@ -231,3 +231,10 @@ Social Buttons
     color: #fff;
 }
 /* jhipster-needle-css-add-main JHipster will add new css style */
+
+/* ==========================================================================
+Create Study Modal
+========================================================================== */
+.email-body{
+    height: 300px;
+}
```
Comment (Nateeo) [inline src/main/webapp/content/css/global.css]: should have newline at end of file

File: src/main/webapp/content/css/global.css [hunk 3]
```diff
@@ -231,3 +231,10 @@ Social Buttons
     color: #fff;
 }
 /* jhipster-needle-css-add-main JHipster will add new css style */
+
+/* ==========================================================================
+Create Study Modal
+========================================================================== */
+.email-body{
```
Comment (Nateeo) [inline src/main/webapp/content/css/global.css]: this class doesn't target anything? Maybe you meant `emailBody`

Comment (jsoulsby) [reply src/main/webapp/content/css/global.css]: Fixed thanks.

File: src/main/webapp/app/entities/study/study-dialog.component.html (hunk 1 above)
Comment (jsoulsby) [reply src/main/webapp/app/entities/study/study-dialog.component.html]: Fixed thanks.

File: src/main/webapp/content/css/global.css [hunk 4]
```diff
@@ -237,4 +237,4 @@ Create Study Modal
 ========================================================================== */
 .email-body{
```
Comment (jsoulsby) [inline src/main/webapp/content/css/global.css]: Have implemented the button navigation as suggested :)

**Comment by Jcen902** [comment]
Can I review them later? I would like to solve my issue first, since I am running a little bit behind

**Comment by Jcen902** [comment]
Nicely done, I think this part is ready

**Comment by jsoulsby** [comment]
@softeng-701 Ready for merge.

**Comment by jsoulsby** [comment]
@softeng-701 Ready to merge.

**Comment by Nateeo** [comment]
@jsoulsby there are merge conflicts that need to be resolved now! 😢

File: src/main/webapp/content/css/global.css (hunk 3 above)
Comment (Nateeo) [reply src/main/webapp/content/css/global.css]: I don't think you've pushed up these changes

**Pull request merged**